config.md

```diff
@@ -265,6 +265,27 @@ Annotations are key-value maps.
 }
 ```
 
+## HypervisorPath
```

Comment:
@jamesodhunt Agree. Configurable default runtime will be good.

I think we can keep `--runtime` daemon flag of containerd as default, so we can explicitly start containerd with some custom runtime e.g. `dockerd --runtime foo`,  then docker can connect to it via `docker --containerd xxxxx`.  If user want to use another runtime, then `docker run --runtime bar`